llvm/include/llvm/ADT/STLExtras.h

```diff
@@ -1535,6 +1535,13 @@ constexpr decltype(auto) makeVisitor(CallableTs &&...Callables) {
   return detail::Visitor<CallableTs...>(std::forward<CallableTs>(Callables)...);
 }
 
+/// Backport of C++23 std::to_underlying.
```

Comment:
Can you write a one-line description of what the function actually does? This being a backport is more of a detail.

Reply:
Done!